crates/kitsune2/tests/blocks.rs

```diff
@@ -707,3 +1055,294 @@ async fn module_messages_of_blocked_peers_are_dropped() {
         .expect("timed out waiting for module message");
     assert_eq!(payload_unblocked, payload_unblocked_received);
 }
+
```

Comment:
And below here are the two blocks tests for outgoing notify and module messages.